packages/KSegmentedControl/KSegmentedControl.spec.js

```diff
@@ -18,7 +18,8 @@ describe('KSegmentedControl', () => {
   })
 
   it('matches snapshot', () => {
-    const wrapper = mount(KSegmentedControl, {propsData: {options: ['1', '2']}})
+    const wrapper = mount(KSegmentedControl, {
+      propsData: { options: ['1', '2'], selected: 'So am I' }})
```

Comment:
so you can pass invalid selected values?

Reply:
I don't think I could add a vue props validator for interdependent props. Fixed this though.